src/modules/assignment/assignment.controller.ts

```diff
@@ -88,9 +87,14 @@ export class AssignmentController {
   }
 
   // Delete a Assignment
-  @Delete('/delete')
-  async deleteAssignment(@Res() res, @Query('uid') uid) {
-    const Assignment = await this.AssignmentService.deleteAssignment(uid);
+  @Delete('/:AssignmentId')
+  async deleteAssignment(
+    @Res() res,
```

Comment:
We will not be using `@Res`. return the object as it is.
Like this: https://github.com/codeforcauseorg/edu-server/commit/2e30df8a0300789d2034b7228086a1f3f4529fdb#r49621473

Reply:
Ok

Reply:
changes made